wasm/src/main/java/com/dylibso/chicory/wasm/types/ValType.java

```diff
@@ -191,6 +207,69 @@ public static int sizeOf(List<ValType> args) {
         return total;
     }
 
+    public ValType substitute(WasmModule context) {
```

Comment:
Is it useful to have the entire `WasmModule` here?
What about using only the `TypeSection`?